js/main.js

```diff
@@ -1,5 +1,6 @@
-import {createTemplateCards} from './generationTemplate.js';
+import {createTemplateCard} from './generationTemplate.js';
 import {createSimilarAdverts} from './data.js';
+import {getElementFromDocument} from './getterGenerationTemplate.js';
 
-createTemplateCards(createSimilarAdverts(), '#map-canvas'); // TODO временно выводим вместо карты
-
+const placeExample = getElementFromDocument('#map-canvas');
+placeExample.append(createTemplateCard(createSimilarAdverts()[0].author, createSimilarAdverts()[0].offer));
```

Comment:
Для демо пойдёт. Можно упростить до такого варианта:
Берём первый элемент массива, а затем его свойства с помощью деструктуризации
```suggestion 
const [{author, offer}] = createSimilarAdverts();
placeExample.append(createTemplateCard(author, offer));
```

Reply:
спасибо, понял все комментарии, постараюсь учитывать эти моменты :)